network/protocol_body.ksy

```diff
@@ -52,7 +52,7 @@ types:
       - id: hdr_ext_len
         type: u1
       - id: body
-        size: hdr_ext_len - 1
+        size: 'hdr_ext_len > 0 ? hdr_ext_len - 1 : 1'
```

Comment:
`hdr_ext_len` is `u1`, so it is always `>=0`. So, your condition brings any difference only for `hdr_ext_len=0`. In this case it sets the size to `1`. Even if for `hdr_ext_len=1` it is set to 0. Is this the effect you are trying to achieve and why do you think that it should be achieved?